smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/ProtocolGenerator.kt

```diff
@@ -85,6 +87,9 @@ interface ProtocolGenerator {
      */
     var serviceErrorProtocolSymbol: Symbol
 
+    val clockSkewProviderSymbol: Symbol
+        get() = DefaultClockSkewProviderSymbol
+
```

Comment:
Here & above is the symbol for the clock skew provider.

In aws-sdk-swift, this property is overridden to provide an AWS-specific implementation.